bootstrap.js

```diff
@@ -230,8 +230,6 @@ function startup(aData, aReason) {
 function shutdown(aData, aReason) {
   // No need to do extra work here
   Log.debug("shutdown, aReason=", aReason);
-  // Load imports for the BOOTSTRAP_REASONS constants; should have happened before in most cases
-  loadImports();
```

Comment:
Why remove this? Was it causing errors?

It may be required in the scenario where Thunderbird starts with the addon disabled, then the addon is uninstalled?